homeassistant/components/coinbase/__init__.py

```diff
@@ -32,46 +29,7 @@
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
 
 
-CONFIG_SCHEMA = vol.Schema(
-    cv.deprecated(DOMAIN),
-    {
-        DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_API_KEY): cv.string,
-                vol.Required(CONF_YAML_API_TOKEN): cv.string,
-                vol.Optional(CONF_CURRENCIES): vol.All(cv.ensure_list, [cv.string]),
-                vol.Optional(CONF_EXCHANGE_RATES, default=[]): vol.All(
-                    cv.ensure_list, [cv.string]
-                ),
-            },
-        )
-    },
-    extra=vol.ALLOW_EXTRA,
-)
-
-
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Set up the Coinbase component."""
-    if DOMAIN not in config:
-        return True
-    ir.async_create_issue(
-        hass,
-        DOMAIN,
-        "remove_yaml",
-        breaks_in_ha_version="2022.12.0",
-        is_fixable=False,
-        severity=ir.IssueSeverity.WARNING,
-        translation_key="removed_yaml",
-    )
```

Comment:
This part needs to stay.

Reply:
I have reset it, but I am confused.
- first, it says `2022.12.0`. Should it not be `2022.11.0` ?
- second, with `cv.removed` will this still trigger?

Reply:
> first, it says 2022.12.0. Should it not be 2022.11.0 ?

I think it can be fully omitted maybe?

> second, with cv.removed will this still trigger?

Good question...